site/content/en/_index.md

```diff
@@ -20,6 +20,19 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
 - `kwok` is the cornerstone of this project, responsible for simulating the lifecycle of fake nodes, pods, and other Kubernetes API resources.
 - `kwokctl` is a CLI tool designed to streamline the creation and management of clusters, with nodes simulated by `kwok`.
 
+### What's the difference with `kubemark`
+
+[kubemark] is a kubelet that does not actually run a container, that behaviors are exactly the same as kubelet,
+so simulating a large number of nodes and pods requires a lot of memory.
+
+`kwok` just simulates the behavior of the node, so it can simulate a large number of nodes and pods with very little memory.
```

Comment:
```suggestion
`kwok`, however, simply simulates the behavior of the node. As a result, it can simulate a large number of nodes and pods using very little memory.
```